ext/drbbdup/drbbdup.dox

```diff
@@ -127,9 +127,9 @@ considers for instrumentation, it also provides a "where" instruction.
 The client must insert code exactly prior to the "where" instruction
 in order to ensure correct instrumentation.
 
-drbbdup also provides drbbdup_is_first_instr() and drbbdup_is_last_instr()
-to determine whether the passed instruction is the first
-or last instruction of a basic block copy respectively.
+drbbdup also provides drbbdup_is_first_instr() drbbdup_is_first_nonlabel_instr()
+and drbbdup_is_last_instr() to determine whether the passed instruction
+is the first or last instruction of a basic block copy respectively.
```

Comment:
Maybe drbbdup.h should do what drmgr.h does:

```
#define drmgr_is_first_instr DO_NOT_USE_drmgr_is_first_instr_USE_drbbdup_instead
```

Reply:
Done.